components/HowItWorks/HowItWorks.tsx

```diff
@@ -107,12 +112,12 @@ function Step({ header, text, subText, animationData, index, isLast }: StepProps
   });
 
   return (
-    <StepWrapper ref={wrapperRef} key={header}>
+    <StepWrapper key={header}>
       <StepNumberWrapper>
         <StepNumber
           initial={{
             backgroundColor: white,
-            color: grey500,
+            color: grey300,
```

Comment:
needed to increase the contrast here because it was failing the accessibility test